packages/dw-react-controls/src/components/SplitPane/SplitPane.types.ts

```diff
@@ -0,0 +1,61 @@
+import { IStyle } from "office-ui-fabric-react/lib/Styling";
+import { IStyleFunctionOrObject } from "office-ui-fabric-react/lib/Utilities";
+
+export interface ISplitPaneProps {
```

Comment:
there's something wrong with <kbd>tab</kbd> and <kbd>space</kbd> formatting here :)